src/account_multisig.ts

```diff
@@ -157,7 +216,34 @@ export class AccountMultisig extends Account {
         return requestId
     }
 
-    async verifyRequestCode(securityCode: string) {
+    async getCodeDefault(): Promise<string> {
+        if (typeof window === 'undefined') {
+            throw new Error('There is no getCodeFallback for Node env. Please provide your own in AccountMultisig constructor options argument.');
+        }
+        const method = await this.get2faMethod();
+        if (!method || !method.kind) {
+            throw new Error('no active 2fa method found');
+        }
+        const code = window.prompt(`Enter security code sent to ${ method.kind.split('2fa-')[1] }`)
+        return code
+    }
+
+    async promptAndVerify() {
+        const method = await this.get2faMethod();
+        const securityCode = await this.getCode(method)
+        try {
+            const { success, res: result } = await this.verifyCode(securityCode);
+            if (!success || result === false) {
+                throw new Error('Request failed with error: ' + JSON.stringify(result));
+            }
+            return typeof result === 'string' && result.length === 0 ? 'true' : result;
```

Comment:
why are we using `'true'`  string as return value?

Reply:
sorry it's cause it was going to be printed, will fix